daemon/datapath.go

```diff
@@ -199,10 +199,17 @@ func (d *Daemon) compileBase() error {
 		return err
 	}
 
+	if option.Config.InstallIptRules {
```

Comment:
We have more places where we do `RemoveRules()` &`InstallRules()`. So perhaps it makes sense to move `TransientRulesStart()` to `RemoveRules()` and introduce the arg `addTransient bool` to `RemoveRules()` (accordingly, `delTransient` to `InstallRules()`).

Reply:
Hmm, could you point out where in master?
```
# git grep -n RemoveRules
daemon/daemon.go:690:   RemoveRules()
daemon/datapath.go:208: d.iptablesManager.RemoveRules()
pkg/datapath/iptables/iptables.go:348:// RemoveRules removes iptables rules installed by Cilium.
pkg/datapath/iptables/iptables.go:349:func (m *IptablesManager) RemoveRules() {
```

Reply:
Ouch, my bad - for some reasons I thought that we call from those places directly `RemoveRules()`, but instead `compileBase()` is being called. So, no need to change anything from your side.